### DISCUSSION TITLE
Windows with 128GB of memory, plotting using the graphics card.

### DISCUSSION THREAD

**Discussion by okwinwin** [Chia-Network/chia-blockchain#16179] [Ideas]
Hello, I'm a small-scale farmer. I don't have advanced server configurations. My computer is just a regular home computer, supporting up to 128GB of memory, with an NVIDIA 2060 Super graphics card. Is it possible to consider that farmers, without large memory, can also use the graphics card for plotting in Chia? Even if it takes a bit longer, that's acceptable. Considering the eco-friendliness of Chia, I hope this can be taken into account. Thank you very, very much.

**Comment by BrandtH22** [comment]
This is currently in development and should be available in the next release version :+1:
